extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -277,7 +277,7 @@ amp-lightbox-gallery .amp-carousel-button {
 .i-amphtml-lightbox-gallery-trans {
   pointer-events: none !important;
   position: fixed !important;
-  z-index: 1001 !important;
+  z-index: 2147483642 !important;
```

Comment:
Should we add documentation or at least a comment to the effect to say that elements with equal or higher z-index will still suffer from this problem until we fix it? Maybe just a comment here saying that this is necessary for fixing this bug here will suffice for now.

Reply:
Added a comment.